docs/user-guide/how-to.md

```diff
@@ -10,6 +10,7 @@ This document describes common tasks you might want to accomplish using HNC.
   * [Create a subnamespace](#use-subns-create)
   * [Inspect namespace hierarchies](#use-inspect)
   * [Propagating policies across namespaces](#use-propagate)
+  * [Limit Resources over parent namespaces](#use-hrq)
```

Comment:
Suggested: `Apply hierarchical resource quotas (HRQs)` (people will be looking for the word "quota" I think)